rep-2000.rst

```diff
@@ -547,6 +547,126 @@ Build System Support:
 - cmake
 - setuptools
 
+Foxy Fitzroy (May 2020 - May 2023)
+----------------------------------
+
+Targeted platforms:
+
++--------------+-----------------------+-------------------------+----------------------+--------------------+---------------+
+| Architecture | Ubuntu Focal (20.04)  | MacOS Catalina (10.15)* | Windows 10 (VS2019)  | Debian Buster (10) | OpenEmbedded /|
+|              |                       |                         |                      |                    | webOS OSE     |
++==============+=======================+=========================+======================+====================+===============+
+|    amd64     |   Tier 1 [d][a][s]    |     Tier 1 [a][s]       |    Tier 1 [a][s]     |     Tier 3 [s]     |               |
++--------------+-----------------------+-------------------------+----------------------+--------------------+---------------+
+|    arm64     |   Tier 1 [d][a][s]    |                         |                      |     Tier 3 [s]     |   Tier 3 [s]  |
++--------------+-----------------------+-------------------------+----------------------+--------------------+---------------+
+|    arm32     |     Tier 2 [d][a][s]  |                         |                      |     Tier 3 [s]     |   Tier 3 [s]  |
++--------------+-----------------------+-------------------------+----------------------+--------------------+---------------+
+
+" * " Tentatively
```

Comment:
Just remove this now, we are going to be on Mojave.

Reply:
6e73ac3